### ISSUE TITLE
Text problems when doing inpainting with stable diffusion 2 api

### ISSUE THREAD

**Issue by AlessioRomanoRobert** [CompVis/stable-diffusion#675]
I am having trouble doing inpainting with stable diffusion 2 using Python gRPC API and model "stable-inpainting-512-v2-0". I am trying to change the background of a product (in this case, a cream) and although the background is changed correctly, the text inside the product is distorted in the final result, even though it is not selected in the mask (the body of cream does not change! only the text inside).

I have checked the mask and it is correctly configured, and I am using the API and algorithm correctly according to documentation. However, I am still having this problem and have not been able to find a solution.

Could you help me understand what might be causing this problem and how I can solve it? Any help or suggestions would be 
greatly appreciated.

ORIGINAL:
![2023-03-24-17-34-19-325bc023-f7f6-4d50-8381-5693057a0eeb](https://user-images.githubusercontent.com/16775494/227609382-6dddf363-bb04-4224-a8bb-8a4fcf603015.png)

MASK:
![mask-2023-03-24-17-34-19-325bc023-f7f6-4d50-8381-5693057a0eeb](https://user-images.githubusercontent.com/16775494/227609444-bae481a3-2d83-4162-af22-b9f37782c612.png)

RESULT:
![completed-1277720669-da28f70d-45fc-4f16-8790-ae26f652ee76](https://user-images.githubusercontent.com/16775494/227609530-00cad6e4-a14f-43be-a59a-65653652882e.png)

**Comment by AlessioRomanoRobert** [comment]
SOLVED: The mask had a color of 254 254 254 and that is why the text changed. stable diffusion support is the best

**Comment by wtliao** [comment]
Hi @AlessioRomanoRobert 
I am doing the similar task as yours. It is more like an out-painting rather than a in-painting task. Could you please let me know the details  to fine tune/train the model, like:
How many training data are used?
Did you fine tune the model resumed from stable diffusion model 512-inpainting-ema.ckpt or 512-base-ema.ckpt base diffusion model, or others?
How many iteration, #GPUs, batch size, and learning rate.

My results after fine tuning are poor. Maybe I can resort some of your experience. Thanks a lot in advanced
